connectors/src/resources/connector/microsoft.ts

```diff
@@ -37,4 +42,10 @@ export class MicrosoftConnectorStrategy implements ConnectorProviderStrategy {
     }
     await resource.delete(transaction);
   }
+
+  async fetchConfigurationsbyConnectorIds(): Promise<
+    Record<ModelId, ConnectorProviderModelResourceMapping["microsoft"]>
+  > {
+    return {};
```

Comment:
We have a configuration for Microsoft, we should probably fetch it here, no?

Reply:
done